docs/rest_api.md

```diff
@@ -208,6 +209,7 @@ Response body (on success):
         "channels": <chanel-id-list>,
         "owner": <project or "admin">,
         "access": <project(s) with access to the network/null>
+        "node, nic": <list of node, nic>
```

Comment:
Maybe change the form of this a bit; the name "node, nic" smells funny, and using a list for this is a little weird. You could rename the field to "connected-nics" or something and make the value a dictionary mapping node -> list of nics, e.g.

```json
...
"connected-nodes": {
    "node-1": ["eth0", "pxe"],
    "node-72": ["eth1"]
}
...
```

Reply:
yeah, I actually wanted to use tuples, but it took me a while to realize that tuples aren't a thing in json.

I'll make this change.